profile/README.md

```diff
@@ -58,7 +57,7 @@ Repositories in status Beta or Production may also occasionally be security audi
 | [kuksa-dds-provider](https://github.com/eclipse-kuksa/kuksa-dds-provider)| Beta | Python-based provider for DDS
 | [kuksa-gps-provider](https://github.com/eclipse-kuksa/kuksa-gps-provider)| Beta | Python-based provider for gpsd
 | [kuksa-someip-provider](https://github.com/eclipse-kuksa/kuksa-someip-provider)| Alpha | Python-based PoC provider for SOME/IP using vsomeip
-
+| [kuksa-mock-provider](https://github.com/eclipse-kuksa/kuksa-mock-provider)| Alpha | Python-based mock provider intended for testing
```

Comment:
Only a nitpick, but "intended for testing" reads like the mock provider is intended to BE tested (also true, it beign alpha and all), but I think the intention was more something like

Python-based provider for mocking VSS signals (to help testing VSS applications)